lib/protobuf/generators/service_generator.rb

```diff
@@ -15,10 +15,10 @@ def compile
       end
 
       def build_method(method_descriptor)
-        name = method_descriptor.name
         request_klass = modulize(method_descriptor.input_type)
         response_klass = modulize(method_descriptor.output_type)
-        "rpc :#{name.underscore}, #{request_klass}, #{response_klass}"
+        name = ENV.key?('PB_RAW_RPC_NAMES') ? method_descriptor.name : method_descriptor.name.underscore
```

Comment:
What about `PB_PRESERVE_RPC_NAMES` or `PB_USE_RAW_RPC_NAMES`? Our env names tend to have an action associated with them.